erpnext/selling/sales_common.js

```diff
@@ -253,7 +253,7 @@ erpnext.selling.SellingController = class SellingController extends erpnext.Tran
 	}
 
 	calculate_commission() {
-		if(!this.frm.fields_dict.commission_rate) return;
+		if(!this.frm.fields_dict.commission_rate || this.frm.doc.docstatus != 0) return;
```

Comment:
```suggestion
		if(!this.frm.fields_dict.commission_rate || this.frm.doc.docstatus === 1) return;
```

Shouldn't this match the server side condition?